runtime/autoload/provider/pythonx.vim

```diff
@@ -39,31 +44,33 @@ function! s:check_version(prog, ver, skip) abort
     return [1, '']
   endif
 
-  let get_version =
-        \ ' -c "import sys; sys.stdout.write(str(sys.version_info[0]) + '.
-        \ '\".\" + str(sys.version_info[1]))"'
-  let min_version = (a:ver == 2) ? '2.6' : '3.3'
-  if system(a:prog . get_version) >= min_version
+  let min_version = (a:ver[0] == 2) ? '2.6' : '3.3'
+  if a:ver >= min_version
```

Comment:
This doesn't make sense. Of course `a:ver` will always be >= `min_version`, because `min_version` has been defined as `a:ver[0]`.